recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
Can you check if the `curl` present in `conda-forge` does have what you need? We are fine pinning software in a case like that, but you are pinning many of them, like `cycler` that is pure Python, and we believe you don't really need that.

Also, see to keep the stack consistent, try to follow `conda-forge`'s pinnings for those that might have an `ABI` change in different version. See https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies